resources/postgres/postgres.go

```diff
@@ -163,3 +197,19 @@ func initDB(pgBinDir, pgDataDir, pgUser string) error {
 
 	return nil
 }
+
+func getBytebaseUser() (int, int, error) {
+	bytebaseUser, err := user.Lookup("bytebase")
```

Comment:
The ways it's written is coupled with the `if currentUser.Username == "root"` logic, I would consider to just inline the method there to avoid its misuse by other places.